docs/release_notes/8.0.6-3.md

```diff
@@ -0,0 +1,96 @@
+* [SERVER-93497](https://jira.mongodb.org/browse/SERVER-93497) - Fixed the issue with using the outdated data by moving user cache invalidation to an `onCommit` handler, created inside of the OpObserver callback, when user's roles get modified
+* [SERVER-97044](https://jira.mongodb.org/browse/SERVER-97044) - Fixed an issue where change streams might incorrectly output a "drop" event during resharding or unsharding of a collection that is or was using zone sharding
+* [SERVER-97860](https://jira.mongodb.org/browse/SERVER-97860) - Fixed the issue with express path returning incorrect results when scanning a unique, multi-field index. The issue occurred when running a query where a predicate is not the `_id` field or a sharded collection has a compound unique index that includes the field used in the query and it doesn't have a unique index. The recommendation is to update to version 8.0.6. Follow closely the upstream recommendations for workaround if you don't wish to upgrade yet 
+* [SERVER-99290](https://jira.mongodb.org/browse/SERVER-99290) - Reduced the strictness of the feature compatibility version check to allow a customer with inconsistent buckets collection to upgrade to FCV 8.0
+* [SERVER-99345](https://jira.mongodb.org/browse/SERVER-99345) - Prevented sharding/moving a time-series buckets collection without the 'timeseries' options on FCV 8.0 and above
+* [SERVER-97911](https://jira.mongodb.org/browse/SERVER-97911) - Fixed the performance regression when MongoDB travers an entire collection before reporting no results instead of doing it immediately when an `
+_id` is given as a hint for a query  
+* [SERVER-100901](https://jira.mongodb.org/browse/SERVER-100901) - Reduce the strictness of an invariant check for the unique ID (uid) in the session identifier (lsid) to match between the operation's context (opCtx) and the incoming command request by treating it as a code error, logging it and have the server continue operating
+
+## Advanced monitoring with debug symbols
+
+With this release, [Percona Server for MongoDB PRO](../psmdb-pro.md) packages and binary tarballs include unstripped binaries. This change enables compatibility with advanced monitoring tools that use debug symbols to provide deeper visibility into the server and to collect detailed diagnostic data and logs.
+
+## Compatibility with Amazon Linux 2023
+
+We build and test Percona Server for MongoDB only on the latest versions of Amazon Linux 2023. Because of the way Amazon Linux updates their libraries, Percona Server for MongoDB 6.0.21-18 is compatible only with Amazon Linux 2023.7.x and won't work on Amazon Linux 2023.6.x and older. 
+
+To upgrade to 6.0.21-18, make sure that you run Amazon Linux 2023.7.x. Use the [update instructions :octicons-link-external-16:](https://docs.aws.amazon.com/linux/al2023/ug/updating.html)
```

Comment:
These paragraphs are duplicated above but the second copy mentions wrong version (6.0.21-18).

Reply:
Fixed